tests/integration_tests/complex_transaction/run.sh

```
@@ -18,14 +18,18 @@ function prepare() {
 
 	# Start 2 CDC servers
-	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "cdc1"
-	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "cdc2" --addr "127.0.0.1:8301"
+	run_cdc_server_with_guard --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "cdc1" --max-restarts 3
+	run_cdc_server_with_guard --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "cdc2" --addr "127.0.0.1:8301" --max-restarts 3
 
 	# Create changefeed
 	cdc_cli_changefeed create --sink-uri="mysql://root@${DOWN_TIDB_HOST}:${DOWN_TIDB_PORT}/"
 }
+function stop_cdc_guards() {
+	stop_cdc_server_guard --workdir $WORK_DIR --logsuffix "cdc1"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)